tests/test_1490-jax-reducers-combinations.py

```diff
@@ -170,455 +30,85 @@ def func_numpyarray_1(x):
 )
 
 
-def test_regular_array_sum_0():
-    def func_regulararray_sum_0(x):
-        return ak.sum(x, 0)
-
-    def func_regulararray_sum_0_jax(x):
-        return jax.numpy.sum(x, 0)
-
-    value_jvp, jvp_grad = jax.jvp(
-        func_regulararray_sum_0, (test_regulararray,), (test_regulararray_tangent,)
-    )
-    value_jvp_jax, jvp_grad_jax = jax.jvp(
-        func_regulararray_sum_0_jax,
-        (test_regulararray_jax,),
-        (test_regulararray_tangent_jax,),
-    )
-
-    value_vjp, vjp_func = jax.vjp(func_regulararray_sum_0, test_regulararray)
-    value_vjp_jax, vjp_func_jax = jax.vjp(
-        func_regulararray_sum_0_jax, test_regulararray_jax
-    )
-
-    assert ak.to_list(value_jvp) == value_jvp_jax.tolist()
-    assert ak.to_list(value_vjp) == value_vjp_jax.tolist()
-    assert ak.to_list(jvp_grad) == jvp_grad_jax.tolist()
-    assert (
-        ak.to_list(vjp_func(value_vjp)[0]) == (vjp_func_jax(value_vjp_jax)[0]).tolist()
-    )
-
+@pytest.mark.parametrize("axis", [0, 1, None])
+@pytest.mark.parametrize("func_ak", [ak.sum, ak.prod, ak.min, ak.max, ak.sum])
+def test_reducer(func_ak, axis):
+    func_jax = getattr(jax.numpy, func_ak.__name__)
```

Comment:
You seem to be removing a large number of tests from this file, but are you actually just consolidating it with `@pytest.mark.parameterize`? I have not verified that all of the old tests still run as parameters of this one test. Is that the case?